profile/README.md

```diff
@@ -10,6 +10,16 @@
   <br>
 </h1>
 
+<h3 align="center">
+  <br>
+  OUR MISSION & VISION
+  <br>
+</h3>
+<h4 align="center">
```

Comment:
add this below discord banner and other banners

Reply:
![image](https://user-images.githubusercontent.com/67006255/167336094-0fa9768e-ba9a-4ddf-941e-94fd30250c36.png)